previews/HST/hst_bundle_previews.sh

```
@@ -0,0 +1,207 @@
+# Collection strip suffixes and WFC3 priority are defined in
+# hst_bundle_browse_map.txt (default: browse_wfc3_drz only).
+#
+# Bundles with no browse_* collections fall back to FITS data collections
+# defined in hst_bundle_fits_map.txt (e.g. hst_14930 → data_stis_x2d).
+#
+# Sample usage:
+#   hst_bundle_previews.sh <path>/hst_16167/hst_16167-deliverable ...
+#   hst_bundle_previews.sh <path>/hst_16167 ...
+################################################################################
+
+HOME=`pwd`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)